src/Server_TEST.cc

```diff
@@ -450,6 +450,42 @@ TEST_P(ServerFixture, RunNonBlocking)
   EXPECT_FALSE(*server.Running(0));
 }
 
+/////////////////////////////////////////////////
+TEST_P(ServerFixture, RunOnceUnpaused)
+{
+    gazebo::Server server;
+    EXPECT_FALSE(server.Running());
+    EXPECT_FALSE(*server.Running(0));
+    EXPECT_EQ(0u, *server.IterationCount());
+
+    // Make the server run fast.
+    server.SetUpdatePeriod(1ns);
+
+    while (*server.IterationCount() < 100)
+        server.RunOnce(false);
+
```

Comment:
You could `EXPECT` the iteration count to be 100 at the end of the loop

Reply:
Done in 1dc665c